transformer/transformer.go

```diff
@@ -176,11 +177,21 @@ func InitTransformers(transformerToInit map[string]string, selector labels.Selec
 	}
 	transformerConfigs := getFilteredTransformers(transformerToInit, selector, logError)
 	transformerNames := []string{}
-	for transformerName := range transformerConfigs {
+	defaultSelectedTransformerNames := []string{}
+	for transformerName, t := range transformerConfigs {
 		transformerNames = append(transformerNames, transformerName)
+		if v, ok := t.ObjectMeta.Labels["move2kube.konveyor.io/default-selected"]; !ok || cast.ToBool(v) {
```

Comment:
Use groupname from types

Reply:
I don't see it in types. It should be there somewhere........

Reply:
done